### PR TITLE
Feature/15 login authentication

### PR DESCRIPTION
**Pull request by RagipGashi**
Login page is done. Please can you review it??
Thank you in advance.

### CODE REVIEW

**Review by RouvenGonzalez — APPROVED** [review]

**Comment by RouvenGonzalez** [comment]
Looks good imo Aproved

File: src/backend-service.test.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+// import { checkEmailAndPassword } from './data-base';
+describe('Testing getEmployee(max_mustermann.id)', () => {
+  it("checkEmainAndPassword('max@mail.com', 'max001'), should return true", () => {
```
Comment (KlugeCh) [inline src/backend-service.test.ts:3]: typo in 'checkEmailAndPassword'
correct all tests

**Review by KlugeCh — COMMENTED** [review]
only typos to fix in test descriptions

File: src/backend-service.test.ts (hunk 1 above)
Comment (RagipGashi) [reply src/backend-service.test.ts:3]: I have corrected the typo in checkEmailAndPassword
Thank you

**Review by RagipGashi — COMMENTED** [review]
Thank you for reviewing @KlugeCh

**Pull request merged**